src/components/Books.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap');
+
+.list {
+  /* width: 10%; */
+
+  /* margin-left: auto;
+  margin-right: auto; */
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+}
+
+.book-list {
+  border-radius: 2px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.book-info {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px;
+}
+
+.add-form {
+  width: 70%;
+  min-height: 10%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.flex-container {
+  display: flex;
+}
+
+.flex-item {
+  margin: 0 0.9rem 0 0;
+}
+
+.title-field {
+  flex-grow: 6;
+}
+
+.category-field {
+  flex-grow: 1;
+}
+
+.add-button {
+  flex-grow: 1;
+}
+
+.book-category {
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.4rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.book-title {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.6rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  font-family: "Roboto Slab", serif;
+  opacity: 0.6;
+  font-size: 1.5rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.category,
+.submit {
+  width: 100%;
+  height: 40px;
+  margin: 0 2px 0 0;
+  padding: 3px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.title-input {
+  width: 100%;
+  height: 2.8rem;
+  margin: 0 0.5rem 0 0;
+  border: solid 3px #e8e8e8;
+}
+
+.category {
+  width: 100%;
+}
+
+.submit {
+  background-color: #0290ff;
+  color: #fff;
+}
+
+.book-atr {
+  display: flex;
+  justify-content: space-between;
+  color: #0290ff;
+}
+
+.btn-primary {
+  background: none;
+  margin: 0;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+  color: #0290ff;
+}
+
+.myform {
+  margin-bottom: 10%;
+}
```

Comment:
[OPTIONAL] - I am marking this as OPTIONAL since the previous CR didn't mention this. However, I highly encourage you to:
- [ ] add a margin on the left so that it looks the same as in the Zeplin design;
- [ ] make the height of your form input field the same;

Please do implement this cause it obviously breaking your app design:pray: